config.json

```diff
@@ -1203,6 +1203,14 @@
           "result"
         ],
         "difficulty": 8
+      },
+      {
+        "slug": "protein-translation",
+        "name": "protein-translation",
```

Comment:
```suggestion
        "name": "Protein Translation",
```
The other fields also need adjusting

Reply:
Oops, usually configlet create adds the space. I should have proofread that closer.